a2/test.ml

```diff
@@ -22,8 +22,37 @@ let data4 : Query.movie list = [
   ("Harry Potter and the Deathly Hallows Part 2","WB",381.01,2011)
 ]
 
+let sort = List.sort compare
+
 let assert_set_equal (a:'a list) (b:'a list) =
-  assert_equal (List.sort compare a) (List.sort compare b)
+  assert_equal (sort a) (sort b)
+
+let ( % ) f g x = f (g x)
```

Comment:
I wonder if it might be more idiomatic to use `|>` instead. (I know it's not identical.)

Reply:
maybe you're right - I just like the pointfree stuff, and `%` is in Batteries, so it's not _totally_ non-idiomatic.

Reply:
Totally -- just read that as me thinking out loud about what's stylistically best.

I do note, though, that our course textbook uses the Batteries alternative, Core -- so it might be worth just keeping an eye on whether there are stylistic differences that could result in an incoherent/inconsistent style. (I have no idea -- just see it as a pitfall.)